app/models/like.rb

```diff
@@ -1,5 +1,5 @@
 class Like < ActiveRecord::Base
-  belongs_to :author, class_name: 'User', foreign_key: :user_id
+  belongs_to :user, class_name: 'User', foreign_key: :user_id
   belongs_to :post
 
```

Comment:
- Also, as requested in the previous code review, kindly ensure to improve upon the following(within the actual models):
- [ ] User model has validations:
- Name must not be blank
- PostsCounter must be an integer greater than or equal to zero.
- [ ] Post model has validations:
- Title must not be blank.
- Title must not exceed 250 characters.
- CommentsCounter must be an integer greater than or equal to zero.

Reply:
Has been resolved.